db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170816203325) do
+ActiveRecord::Schema.define(version: 20170814174637) do
```

Comment:
Not sure why this shows up here. You should only introduce changes in db/schema.rb if you have a migration. Otherwise something has gone wrong. Make sure you `git pull` in your master branch, then go to your branch and rebase. In any case these changes, should **not** be here.

Reply:
yes. Thanks i will do it. :smile: